android/src/main/cpp/NativeProxy.cpp

```diff
@@ -348,19 +349,41 @@ void NativeProxy::installJSIBindings(
 
   std::weak_ptr<ErrorHandler> weakErrorHandler = errorHandler;
 
+  auto convertToArray = [](jsi::Runtime &rt, const std::string &value) {
+    std::vector<float> transformMatrixList;
+    std::istringstream stringStream(value);
+    std::copy(
+        std::istream_iterator<float>(stringStream),
+        std::istream_iterator<float>(),
+        std::back_inserter(transformMatrixList));
+    assert(transformMatrixList.size() == 9);
+    jsi::Array matrix(rt, 9);
+    for (int i = 0; i < 9; i++) {
+      matrix.setValueAtIndex(rt, i, transformMatrixList[i]);
+    }
+    return matrix;
+  };
+
   layoutAnimations->cthis()->setAnimationStartingBlock(
-      [wrt, weakModule, weakErrorHandler](
+      [wrt, weakModule, weakErrorHandler, convertToArray](
           int tag,
           alias_ref<JString> type,
           alias_ref<JMap<jstring, jstring>> values) {
         auto &rt = *wrt.lock();
         jsi::Object yogaValues(rt);
         for (const auto &entry : *values) {
```

Comment:
Is it possible to use this syntax here?
```suggestion
        for (const auto &[first, second] : *values) {
```

Reply:
I personally don't see a reason to, when both are immediately converted to something else.